src/Giraffe/ComputationExpressions.fs

```diff
@@ -1,30 +1,31 @@
-/// **Description**
-///
+/// <summary>
 /// A collection of F# computation expressions:
 ///
 /// `opt {}`: Enables control flow and binding of Option<'T> objects
 /// `res {}`: Enables control flow and binding of Result<'T, 'TError> objects
-///
+/// </summary>
 module Giraffe.ComputationExpressions
 
+/// <summary>
+/// Enables control flow and binding of `Option<'T>` objects
+/// </summary>
+/// <param name="unit"></param>
+/// <returns></returns>
 type OptionBuilder() =
     member __.Bind(v, f)   = Option.bind f v
     member __.Return v     = Some v
     member __.ReturnFrom v = v
     member __.Zero()       = None
 
-/// **Description**
-///
-/// Enables control flow and binding of `Option<'T>` objects
-///
 let opt = OptionBuilder()
 
+/// <summary>
```

Comment:
moved the docs to the type - seems to make more sense but could be missing something